ml-agents/mlagents/trainers/learn.py

```diff
@@ -509,13 +508,6 @@ def create_environment_factory(
     env_args: Optional[List[str]],
     log_folder: str,
 ) -> Callable[[int, List[SideChannel]], BaseEnv]:
-    if env_path is not None:
```

Comment:
I think this was here so that passing an invalid environment path would exit with a non-zero return code. Is that still the case?

Reply:
It seemed to me that is was a duplicate of the error raising in execulable_launcher. I am usually against having multiple errors trying to catch the same thing.
I tested on my machine and the error was raised, does the error need to be thrown here ?

Reply:
@vincentpierre @chriselion this was originally to fix the non-zero exit code, but subsequently other changes we have made to exception handling have made it irrelevant.  Now I believe exceptions will be correctly bubbled up from subprocess environments.

Reply:
Sounds good.